bin/ungit

```diff
@@ -14,7 +14,11 @@ const bugtracker = new BugTracker('launcher');
 // Fastest way to find out if a port is used or not/i.e. if ungit is running
 const net = require('net');
 const server = net.createServer();
-const cleanExit = () => process.exit();
+let child;
+const cleanExit = () => {
+  child.kill('SIGINT');
```

Comment:
Please add a check if child is not undefined as it might be possible to kill the process before the child is spawned.

Reply:
Right